.travis.yml

```diff
@@ -21,9 +24,14 @@ install:
 script:
   test/run_tests.sh $PWD/testing-dependencies/mod_pagespeed/ \
                     $PWD/nginx/sbin/nginx
+dist: trusty
 sudo: required
 compiler:
   - gcc
 notifications:
   email:
+    - cheesy@google.com
```

Comment:
Do you think it makes sense to create a group for this? pagespeed-ci or something?